.github/workflows/ci.yml

```diff
@@ -0,0 +1,36 @@
+name: CI
+
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+      - master
```

Comment:
Nitpick, but this repo doesn't use a `master` branch.

Reply:
removed! :partying_face: